cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -567,10 +568,19 @@ public Service generateExternalBootstrapService() {
             ports = Collections.singletonList(createServicePort(EXTERNAL_PORT_NAME, EXTERNAL_PORT, EXTERNAL_PORT,
                 nodePort, "TCP"));
 
+            Map<String, String> dnsAnnotations = Collections.EMPTY_MAP;
```

Comment:
`Collections.emptyMap()` will avoid a compiler warning about raw types.

Reply:
I didn't know that ... should be fixed now. Thanks.